bootstrap

```diff
@@ -47,16 +47,61 @@ get_files() ->
 
     %% include all compiled modules and the .app file
     Beams = [filelib:fold_files(Dir, "\\.app$|\\.beam$", true, Fun, [])
-	     || Dir <- Apps],
+             || Dir <- Apps] ++ elixir_beams(),
 
     %% include all contents of the priv dirs
     %% except for .js.map files, which are heavy but unnecessary
     Privs = [filelib:fold_files(filename:join([Dir, "priv"]),
                                 "^((?!\\.js\\.map).)*$", true, Fun, [])
-	     || Dir <- Apps],
+             || Dir <- Apps],
 
     lists:flatten([Beams, Privs]).
 
 file_contents(Filename) ->
     {ok, Bin} = file:read_file(Filename),
```

Comment:
Maybe even call beam_lib:strip/1 here to ensure future plugins are always stripped.